lambdas/thumbnail/index.py

```diff
@@ -249,39 +254,52 @@ def lambda_handler(request):
     input_ = request.args.get('input', 'image')
     output = request.args.get('output', 'json')
     page = request.args.get('page', 1)
```

Comment:
@akarve the page should be parsed, otherwise there's an error:
```
'2' is not of type 'integer'

Failed validating 'type' in schema['properties']['page']:
    {'minimum': 1, 'type': 'integer'}

On instance['page']:
    '2'
```